Makefile

```diff
@@ -19,10 +19,10 @@ build:  ## build the XBlock image
 	$(COMPOSE) build lms
 .PHONY: build
 
-config/settings.yml:
+config/settings:
```

Comment:
I think changing the name would break things as it much match the name of the file.

Reply:
this target is required just above by `dev` but it does not include `.yml` extension

Reply:
Please add the .yml instead, because this rule has to match the file name.